decimal/decimal.go

```diff
@@ -102,6 +102,12 @@ func (d Decimal) Float64() float64 {
 	return float64(d.i.Int64()) * math.Pow(float64(10), float64(-d.scale))
 }
 
+// Value returns the value of the decimal in units of its scale. For example,
+// the decimal 0.250 with scale 3 will return a big.Int with value 250.
+func (d Decimal) Value() *big.Int {
```

Comment:
What is the use case for this? Seems not useful to get the coefficient without the scale?

Reply:
Conversion to D8. Could use .Float64() but that will potentially introduce errors. Can call ToScale() before calling Value() to specify the scale.